homeassistant/components/logbook/models.py

```diff
@@ -83,17 +77,17 @@ def __init__(
     @property
     def context_id(self) -> str | None:
         """Return the context id."""
-        return bytes_to_ulid_or_none(self.context_id_bin)
+        return bytes_to_ulid_or_none(self.row.context_id_bin)
 
     @property
     def context_user_id(self) -> str | None:
         """Return the context user id."""
-        return bytes_to_uuid_hex_or_none(self.context_user_id_bin)
+        return bytes_to_uuid_hex_or_none(self.row.context_user_id_bin)
 
     @property
     def context_parent_id(self) -> str | None:
         """Return the context parent id."""
-        return bytes_to_ulid_or_none(self.context_parent_id_bin)
+        return bytes_to_ulid_or_none(self.row.context_parent_id_bin)
```

Comment:
We currently don't call these in any of the logbook platforms so I could remove them but I left them for backwards compat in case some custom component is using them